inc/reminder.class.php

```diff
@@ -1249,4 +1241,7 @@ function getRights($interface = 'central') {
       return $values;
    }
 
+   function getItilObjectItemType() {
```

Comment:
This looks like a hack.

Reply:
getAlreadyPlannedInformations calls getItilObjectItemType